### PR TITLE
Typography - Incremental version demo

### PR DESCRIPTION
**Pull request by halocline**
[Deploy Preview](https://deploy-preview-3113--keen-mayer-a86c8b.netlify.app/)

#### What does this PR do?

Staging modified type proposal for https://github.com/grommet/hpe-design-system/issues/3102

### CODE REVIEW

**Comment by taysea** [comment]
In order to better assess some of the heading/paragraph relationships, I wonder if it would be valuable to sift out usage of `size` on Heading implementations such as "Cards" and other key areas? 

For example, this card uses `size="small"` on Heading I believe, so it doesn't give a realistic way to assess the proposed changes (aside from highlighting that teams will need to be sifting this out as well)

Screen Shot 2023-01-19 at 9 50 53 AM

File: aries-site/src/themes/aries.js [hunk 1]
```diff
@@ -16,34 +17,599 @@ export const aries = deepMerge(hpe, {
...
+    large: {
+      size: '22px',
+      height: '26px',
+    },
+    xlarge: {
+      size: '28px',
+      height: '32px',
+    },
+    xxlarge: {
+      size: '32px',
```
Comment (taysea) [inline aries-site/src/themes/aries.js:304]: The latest "typography translation" has paragraph xl as size 30px, height: 36px

File: aries-site/src/themes/aries.js [hunk 2]
```diff
@@ -16,34 +17,599 @@ export const aries = deepMerge(hpe, {
...
+    medium: {
+      size: '18px',
+      height: '24px',
+    },
+    large: {
+      size: '22px',
+      height: '26px',
+    },
+    xlarge: {
+      size: '28px',
```
Comment (taysea) [inline aries-site/src/themes/aries.js:300]: Double check all of these values, latest proposed from Chris are slightly different: https://www.figma.com/file/wtwk7YovJMvDb3cfWCMWpu/HPE-Product-Brand-Guidelines?node-id=119%3A5816&t=M5EfR2WJoJkBWzIv-4

**Comment by britt6612** [comment]
one area I found:
https://deploy-preview-3113--keen-mayer-a86c8b.netlify.app/components/card#misuses
Screen Shot 2023-01-30 at 10 21 01 AM

**Comment by halocline** [comment]
Demo branch no longer needed.

**Pull request closed without merging**